include/mqtt/mqtt_enc28j60_ethernet.h

```diff
@@ -0,0 +1,37 @@
+#ifndef __MQTT_ETH_H__
+#define __MQTT_ETH_H__
+
+#include <EthernetENC.h>
+
+#include <inttypes.h>
+
+#include "mqtt.h"
```

Comment:
shouldn't this be mqtt/mqtt.h?

Reply:
it will link to the mqtt.h in the same folder